robot/EDA/tests/browser/settings/eda_settings_system.robot

```diff
@@ -2,14 +2,15 @@
 
 Resource        robot/EDA/resources/EDA.robot
 Suite Setup     Open Test Browser
-Suite Teardown  Delete Records and Close Browser
+Suite Teardown  Capture screenshot and delete records and close browser
 
 *** Test Cases ***
 
 Test EDA System Settings
-    Go To Eda Settings
-    Wait For Locator                    eda_settings.system_tab
-    Click On Element                    eda_settings.system_tab
+    [Setup]  Run keywords
+    ...  Go to EDA Settings
+    ...  AND    Wait For Locator        eda_settings.system_tab
+    ...  AND    Click On Element        eda_settings.system_tab
```

Comment:
curious about why this particular test case has setup? is there any added advantage?

Reply:
This is something Jess and Bryan have decided to do because it helps to differentiate between the setup and the actual test. This code change is actually part of another PR but that hasn't been merged yet due to code freeze. Hence showing up in the diff here.
Take a look at Bryan's comments here: https://github.com/SalesforceFoundation/EDA/pull/1150/files#diff-08c0ef73af989b71d63216db7ea44ce3R16